src/main/java/com/sapozhnykov/dao/cart/impl/CartDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.sapozhnykov.dao.cart.impl;
+
+import com.sapozhnykov.dao.cart.CartDao;
+import com.sapozhnykov.domain.Product;
+
+import java.util.ArrayList;
+
+public class CartDaoImpl implements CartDao {
+    private ArrayList<Product> cartProducts = new ArrayList<>();
```

Comment:
better to use interface instead class for variable type

Reply:
At the last lesson, Artem said that we would not have a Cart. I removed it from the project.